### PR TITLE
fix: drop logging packages hard-coded

### PR DESCRIPTION
**Pull request by mfrw**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Clean up hardcoded logging for packages in favor of a dynamically generated list which is already there

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Comment by hbeberman** [comment]
LGTM!

I'd like @alexeldeib to take a quick look too before merging.

**Review by hbeberman — APPROVED** [review]

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -241,20 +209,8 @@ fi
 ls -ltr /opt/gpu/* >> ${VHD_LOGS_FILEPATH}
 
 installBpftrace
-echo "  - bpftrace" >> ${VHD_LOGS_FILEPATH}
-
-cat << EOF >> ${VHD_LOGS_FILEPATH}
-  - nvidia-docker2=${NVIDIA_DOCKER_VERSION}
-  - nvidia-container-runtime=${NVIDIA_CONTAINER_RUNTIME_VERSION}
-  - nvidia-gpu-driver-version=${GPU_DV}
-  - nvidia-fabricmanager=${GPU_DV}
```
Comment (alexeldeib) [inline vhdbuilder/packer/install-dependencies.sh:250]: this one comes from L210 a bit up, could we change it to e.g. "nvidia-driver=${NVIDIA_DRIVER_IMAGE_TAG}"

this is a container we build ourselves but the tag contains the driver version

we can delete all these 4 like you've done, but would be great to replace

File: vhdbuilder/packer/install-dependencies.sh [hunk 2]
```diff
@@ -241,20 +209,8 @@ fi
 ls -ltr /opt/gpu/* >> ${VHD_LOGS_FILEPATH}
 
 installBpftrace
-echo "  - bpftrace" >> ${VHD_LOGS_FILEPATH}
```
Comment (alexeldeib) [inline vhdbuilder/packer/install-dependencies.sh:244]: this also doesn't come from os packages, might be easier to run `bpftrace -version` or something

File: vhdbuilder/packer/install-dependencies.sh [hunk 3]
```diff
@@ -241,20 +209,8 @@ fi
...
-
-cat << EOF >> ${VHD_LOGS_FILEPATH}
-  - nvidia-docker2=${NVIDIA_DOCKER_VERSION}
-  - nvidia-container-runtime=${NVIDIA_CONTAINER_RUNTIME_VERSION}
-  - nvidia-gpu-driver-version=${GPU_DV}
-  - nvidia-fabricmanager=${GPU_DV}
-EOF
 
 installBcc
-cat << EOF >> ${VHD_LOGS_FILEPATH}
```
Comment (alexeldeib) [inline vhdbuilder/packer/install-dependencies.sh:254]: similarly, not from OS packages. you could pull the version var out of the install function and log it here, or avoid touching this one :)

File: vhdbuilder/packer/install-dependencies.sh (hunk 1 above)
Comment (mfrw) [reply vhdbuilder/packer/install-dependencies.sh:250]: @alexeldeib PTAL :)

**Review by alexeldeib — APPROVED** [review]
lgtm, thanks and welcome ! 🙂

**Pull request merged**